CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## 1.0.0 (18.07.2021)
+* First allstar configuration with Opt Out Strategy.
```

Comment:
@CristhianRodriguezMolina @SanGeeky Could you please clarify? What about something like `Allstar config for the kommit's OSPO`